cljfmt/src/cljfmt/core.cljc

```diff
@@ -341,6 +341,25 @@
   ([form indents alias-map]
    (transform form edit-all should-indent? #(indent-line % indents alias-map))))
 
+(defn split-maps
+  "Puts each hashmap pair on it's own line"
+  [form]
+  (transform form edit-all z/map?
+             (fn [zloc]
+               (let [split
+                     (loop [z (zip/children zloc) i 1 accumulate []]
+                       (let [nodes (if (= 0 (mod i 4))
+                                     ; 4th elements are the whitespace between pairs
```

Comment:
Can we guarantee the 4th element is the whitespace between pairs? What about a map like:

```clojure
{:x 1 ;; some comment
 :y 2}
```